Mathlib/Data/Finset/Pointwise.lean

```diff
@@ -2208,8 +2241,57 @@ protected theorem neg_smul [DecidableEq α] : -s • t = -(s • t) := by
 
 end Ring
 
+section BigOps
+section CommMonoid
+variable [CommMonoid α] {ι : Type*} [DecidableEq ι]
+
+@[to_additive (attr := simp)] lemma prod_inv_index [InvolutiveInv ι] (s : Finset ι) (f : ι → α) :
+    ∏ i in s⁻¹, f i = ∏ i in s, f i⁻¹ := prod_image $ inv_injective.injOn _
```

Comment:
I don't remember whether we have fixed the coding style about `$` vs `<|`. Do you?

Reply:
People sometimes say to use `<|` but it's not very applied, mostly because `|>` is weird and `$` is shorter and less ASCII art.